probe-dictionary/index.html

```diff
@@ -31,11 +31,11 @@
       <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
-      <a class="navbar-brand" href="#">Probe dictionary</a>
+      <a class="navbar-brand" href="#">Probe Dictionary</a>
 
       <div class="collapse navbar-collapse" id="navbarCollapse">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item active">
+        <ul class="navbar-nav mr-auto navbar-custom">
```

Comment:
What is the purpose of `navbar-custom` here? It appears to add no styles or semantic information... but I'm not the most familiar with bootstrap, so maybe it's customary?

Reply:
@chutten That was my mistake, a carry over from the last custom style. I have promptly removed.